### ISSUE TITLE
Better Support for Keeping Figures Close to their Anchor Points

### ISSUE THREAD

**Issue by fortes** [Treesaver/treesaver#164]
Figures are currently aggressively pulled into pages, even if their anchor point occurs much later in markup.

**Comment by stiller** [comment]
Yes, this is pretty important to us, since relevance is dependent on position for figures.

**Comment by mjc-gh** [comment]
I would like this see this as well. Thanks

**Comment by pph7** [comment]
Any progress on this?
